test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/StoreTransactionTests.cs

```diff
@@ -452,6 +452,17 @@ public async Task GivenInstanceWithPatientIDWithInvalidChars_WhenStoreInstanceWi
         failedAttributesSequence.Items[0].GetString(DicomTag.ErrorComment));
     }
 
+    [Fact]
+    public async Task GivenAValidUpdateAttributeRequest_WhenUpdating_ThenShouldSucceed()
+    {
+        List<string> studyInstanceUIDs = new List<string> { TestUidGenerator.Generate(), TestUidGenerator.Generate() };
+        DicomDataset dataset = new DicomDataset(new DicomPersonName(DicomTag.PatientBirthName, "foo"));
+
+        using DicomWebResponse response = await _instancesManager.UpdateAsync(studyInstanceUIDs, dataset);
```

Comment:
Where do you turn on this feature for E2E tests?

Reply:
Good catch. I have added it in `InProcTestDicomWebServer`.